dev/tools/localization/bin/gen_localizations.dart

```diff
@@ -76,6 +76,8 @@ String generateArbBasedLocalizationSubclasses({
   assert(supportedLanguagesConstant.isNotEmpty);
   assert(supportedLanguagesDocMacro.isNotEmpty);
 
+  bool isNbSynonymOfNo = false;
```

Comment:
Maybe I can come up with a better variable name. It reads oddly when I say it out loud

Reply:
The name seems OK (we're deep in a corner here) but a comment that referred to #53036 would be helpful